### DISCUSSION TITLE
Could someone please send me a slack invite?

### DISCUSSION THREAD

**Discussion by caozhuozi** [milvus-io/milvus#27658] [Q&A and General discussion] [answered]
![image](https://github.com/milvus-io/milvus/assets/68834160/d9e6ee3a-4bdc-49c7-8782-7bfeb63f2a62)
link: https://milvus.io/slack/

I already sent 3 invite requests before, but got no responce. Could someone check the invite request from wxdahk@gmail.com ?

This time seems the invite request web link dosnt work. It shows "loading"

Thanks...

**Comment by yhmo** [comment]
I have sent an invite email to you from slack, please check your email box.

**Reply by caozhuozi** [reply]
@yhmo Thanks for your kind quick responce. After click the button in the email. I got this alert:
![image](https://github.com/milvus-io/milvus/assets/68834160/a61390a4-fbcf-496e-9ee9-382d46c2c228)
Could you please also tell me what can I do from my side？

**Reply by yhmo** [reply]
Seems it no longer supports new signups from China.

**Reply by yhmo** [reply, marked as answer]
Doesn't matter, you can put your questions in [here](https://github.com/milvus-io/milvus/discussions), or bugs in [issues](https://github.com/milvus-io/milvus/issues).

**Reply by caozhuozi** [reply]
Thanks for the information. I just used VPN to swich my region. Now I can succesfully login into the workspace. Thanks!

**Comment by caozhuozi** [comment]
> Seems it no longer supports new signups from China.
